packages/block-library/src/post-navigation-link/block.json

```
@@ -28,6 +28,16 @@
 		"arrow": {
 			"type": "string",
 			"default": "none"
+		},
+		"excludedTerms": {
+			"type": "string"
```

```suggestion
			"type": [ "string", "array" ]
```

It looks like `excludedTerms` can also be an array, should we include this in the accepted types here?

---

Should this only apply as to what the attribute of the block allows, now to what the `get_next_post` WP function allows? In that case if the code always works with strings (coming from the inspector input) no reason to support array I think.

---

Ahh ok, that makes sense. I guess multiple values would be comma-separated in the inspector input string then.